code/modules/atmospherics/machinery/components/trinary_devices/mixer.dm

```diff
@@ -122,8 +129,8 @@
 	data["on"] = on
 	data["set_pressure"] = round(target_pressure)
 	data["max_pressure"] = round(MAX_OUTPUT_PRESSURE)
-	data["node1_concentration"] = round(node1_concentration*100)
-	data["node2_concentration"] = round(node2_concentration*100)
+	data["node1_concentration"] = round(node1_concentration*100, 1)
+	data["node2_concentration"] = round(node2_concentration*100, 1)
```

Comment:
Reason for this?

Reply:
Round(x) is actually a floor operation, so this leads to percentages sometimes not adding to 100. I should probably add that to the changelog, actually.